yb-voyager/cmd/importData.go

```diff
@@ -177,7 +177,18 @@ func startExportDataFromTargetIfRequired() {
 		utils.ErrExit("could not find yb-voyager - %w", lookErr)
 	}
 	env := os.Environ()
-	env = append(env, fmt.Sprintf("TARGET_DB_PASSWORD=%s", tconf.Password))
+	found := false
+	for i, envVar := range env {
+		if strings.HasPrefix(envVar, "TARGET_DB_PASSWORD=") {
+			// Change the value of TARGET_DB_PASSWORD
+			env[i] = "TARGET_DB_PASSWORD=" + tconf.Password
+			found = true
+			break
+		}
+	}
+	if !found {
+		env = append(env, "TARGET_DB_PASSWORD="+tconf.Password)
+	}
```

Comment:
Can you please test this for `SOURCE_DB_PASSWORD` and `SOURCE_REPLICA_DB_PASSWORD`, we might have to add this at all places as I believe we are not handling this at other places.

Reply:
Added similar code for `SOURCE_DB_PASSWORD`. Couldn't find any code where `SOURCE_REPLICA_DB_PASSWORD` is being appended to the env variable.